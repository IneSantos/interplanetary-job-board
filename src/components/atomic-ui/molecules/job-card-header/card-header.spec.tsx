import React from "react";
import { render } from '@testing-library/react';
import CardHeader from "./card-header";

describe("CardHeader atom tests", () => {
  test(`CardHeader component render`, () => {
    const { container } = render(
      <CardHeader position="This is a position" company="This is a company" location="This is a location" contract="This is a contract"/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
