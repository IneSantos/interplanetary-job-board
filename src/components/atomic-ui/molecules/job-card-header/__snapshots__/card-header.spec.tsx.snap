// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader atom tests CardHeader component render 1`] = `
<div
  class="card-header"
>
  <h2
    class="title"
  >
    This is a position
  </h2>
  <div
    class="job-details"
  >
    <div
      class="job-detail job-detail-company"
    >
      <i
        class="fas fa-building icon"
      />
      <span>
        This is a company
      </span>
    </div>
    <div
      class="job-detail job-detail-location"
    >
      <i
        class="fas fa-map-marker-alt icon"
      />
      <span>
        This is a location
      </span>
    </div>
    <div
      class="job-detail job-detail-contract"
    >
      <i
        class="fas fa-file-signature icon"
      />
      <span>
        This is a contract
      </span>
    </div>
  </div>
</div>
`;
